styles/navigation.css

```diff
@@ -0,0 +1,35 @@
+@import url(/styles/utility.css);
+@import url(/styles/index.css);
+
+.c-navbar {
+  background-color: var(--primary-color);
+  min-height: 5rem;
+  width: 100%;
+}
+
+.c-navbar a {
+  color: var(--secondary-color);
+  text-decoration: none;
+}
+
+.c-navbar ul li {
+  display: inline-block;
+}
+
+.nav-input {
```

Comment:
Just a small suggestion, maybe you can use `navbar-input` here. It makes it more consistent.

Reply:
Thanks for the suggestion, changed the name